client/src/plus/ai-help/use-ai.ts

```diff
@@ -384,6 +384,7 @@ export function useAiChat({
       eventSourceRef.current?.close();
       eventSourceRef.current = undefined;
       setLoadingState("failed");
+      mutate("/api/v1/plus/ai/help/quota");
```

Comment:
Nit: Shall we extract this constant here:

https://github.com/mdn/yari/blob/69395528f0943a99cc0fd033407b72830fb58f4f/client/src/plus/common/api.tsx#L1-L3

Reply:
Good point!